cfn.yaml

```diff
@@ -49,10 +51,144 @@ Resources:
               Action:
                 - lambda:InvokeFunction
               Resource: "*"
-  Lambda:
+        - PolicyName: sqs
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action:
+                - sqs:DeleteMessage
+                - sqs:DeleteMessageBatch
+                - sqs:ReceiveMessage
+                - sqs:SendMessage
+                - sqs:SendMessageBatch
+              Resource:
+                - !GetAtt NewsletterCutOffDateQueue.Arn
+                - !GetAtt CleanseListQueue.Arn
+
+  GetCutOffDatesLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Sub newsletter-get-cut-off-dates-${Stage}
```

Comment:
Could be even more explicit in the names of the lambdas and call them `newsletter-cleanse-...`